### PR TITLE
nfs changes

### PR DESCRIPTION
**Pull request by kofemann**

### CODE REVIEW

File: release-notes-3.0 [hunk 1]
```diff
@@ -637,6 +637,35 @@ property. dCache now recognizes this property and uses it in space
...
 file does not match the specified size, the file is marked as broken.
 
+#### nfs: autodiscovery of nfs4domain for idmapping
+
+For correct user id mapping nfs4 requires that server and client use
+the same naming scope, called nfs4domain. This implies a consistent
+configuration on both sides. To lower deployment overhead a special
+auto-discovery mechanism was introduced by SUN Microsystems - a DNS
+TXT record. Staring from version 3.0 dcache supports this discovery
+mechanism. When `nfs.domain` property is set, it get used. If it's
```
Comment (gbehrmann) [inline release-notes-3.0]: get -> gets

File: release-notes-3.0 [hunk 2]
```diff
@@ -637,6 +637,35 @@ property. dCache now recognizes this property and uses it in space
...
+left unset, then DNS TXT record for `_nfsv4idmapdomain` is taken or
+the defult `localdomain` is used when DNS record is absent.
```
Comment (gbehrmann) [inline release-notes-3.0]: defult -> default

File: release-notes-3.0 [hunk 3]
```diff
@@ -637,6 +637,35 @@ property. dCache now recognizes this property and uses it in space
...
+file by the same user coming from the same host. This should improve
+compatibility with existing clients as they expect such behavior which
+is allowed by nfs protocol specification.
+
+#### nfs: added support for flexfile file layout
+
+Flexfile layout type aims to be an extension to pNFS to allow the use of
+storage devices in a fashion such that they require only a quite limited
+degree of interaction with the metadata server. The RHEL 7.2 and it's
+derivatives are providing an experimental support for flexfile layout
```
Comment (gbehrmann) [inline release-notes-3.0]: Drop 'an'

**Review by gbehrmann — CHANGES_REQUESTED** [review]
Also please remove lines 757 and 758 (place holder bullets for nfs changes).

File: release-notes-3.0 (hunk 3 above)
Comment (kofemann) [reply release-notes-3.0]: fixed

File: release-notes-3.0 [hunk 4]
```diff
@@ -574,8 +574,8 @@ fairly easy to hook alternate storage backends into dCache.
 #### pool: sorted listing of movers
 
 In the _nice to have_ category, dCache 3 supports options in the
-`mover ls` admin shell command to sort the output by last access time
-and file size.
+`mover ls` admin shell command to sort the output by mover's last
+access time and transfered data size.
```
Comment (gbehrmann) [inline release-notes-3.0]: transfered -> transferred

Comment (kofemann) [reply release-notes-3.0]: fixed

**Review by gbehrmann — APPROVED** [review]

**Pull request merged**